src/shared/rdf-model-util.ts

```diff
@@ -100,4 +101,12 @@ export class RdfModelUtil {
     static appendLocale(value: string, locale: string): string {
         return `${value} ${locale ? `@${locale}` : ''}`;
     }
+
+    static throwErrorIfUnsupportedVersion(rdfModel: RdfModel): void {
+        if (rdfModel && KnownVersion.isVersionSupported(rdfModel.getMetaModelVersion()) === false) {
+            throw Error(
+                `Version ${rdfModel.getMetaModelVersion()} is not supported by this versions. Supported versions are: ${KnownVersion.getSupportedVersions().join()}`
```

Comment:
SAMM ${rdfModel.getMetaModelVersion()} is not supported by this project. Supoorted versions are: ${KnownVersion.getSupportedVersions().join(',')}.

Reply:
done